<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElkaMainWindow</class>
 <widget class="QMainWindow" name="ElkaMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ElkaMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tab_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="Command">
     <attribute name="title">
      <string>Command</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>401</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QSpinBox" name="gains_kpp">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>44</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="gains_label">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>270</y>
         <width>201</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Gains: kpp,kip,kdp,kpr,kir,kdr,kpy</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="gains_kip">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>290</y>
         <width>44</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="gains_kdp">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>290</y>
         <width>44</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="gains_kpr">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>290</y>
         <width>44</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="gains_kir">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>290</y>
         <width>44</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="gains_kdr">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>290</y>
         <width>44</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="gains_kpy">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>290</y>
         <width>44</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="input_mode_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>71</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Input Mode</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="joystick_input_button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>105</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Joystick</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="keyboard_input_button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>105</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Keyboard</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="manual_input_line">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>781</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="manual_input_label">
       <property name="geometry">
        <rect>
         <x>338</x>
         <y>330</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Manual Command</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stop_elka_button">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>80</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop Elka</string>
       </property>
      </widget>
      <widget class="QPushButton" name="start_elka_button">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>20</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Elka</string>
       </property>
      </widget>
      <widget class="QPushButton" name="properties_button">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>140</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Properties</string>
       </property>
      </widget>
      <widget class="QSlider" name="thrust_slider">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>18</width>
         <height>160</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="roll_slider">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>50</y>
         <width>18</width>
         <height>160</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="pitch_slider">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>50</y>
         <width>18</width>
         <height>160</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="yaw_slider">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>50</y>
         <width>18</width>
         <height>160</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="input_cmds_label">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Inputs: r,p,t,y</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Plot">
     <attribute name="title">
      <string>Plot</string>
     </attribute>
     <widget class="QPushButton" name="parse_log_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Parse Log</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="data_sets_tree">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>281</width>
        <height>191</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Current Data Sets</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>Inputs</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Outputs</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gains</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Acks</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Drops</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="save_log_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Log</string>
      </property>
     </widget>
     <widget class="QPushButton" name="export_data_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Export Data</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plot_data_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot Data</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Editor">
     <attribute name="title">
      <string>Editor</string>
     </attribute>
     <widget class="QTextBrowser" name="scripts_text_browser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>401</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="undoRedoEnabled">
       <bool>true</bool>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::WidgetWidth</enum>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open_2"/>
    <addaction name="action_Save_As"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Exit_3"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Paste"/>
    <addaction name="action_Paste_2"/>
   </widget>
   <widget class="QMenu" name="menu_Run">
    <property name="title">
     <string>&amp;Run</string>
    </property>
    <widget class="QMenu" name="menuInput_Mode">
     <property name="title">
      <string>Input Mode</string>
     </property>
     <addaction name="action_Joystick"/>
     <addaction name="action_Keyboard"/>
    </widget>
    <addaction name="menuInput_Mode"/>
    <addaction name="action_Set_Gains"/>
    <addaction name="separator"/>
    <addaction name="actionStart_Elka"/>
    <addaction name="action_Stop_Elka"/>
    <addaction name="separator"/>
    <addaction name="actionProperties_2"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Help_Dialogs"/>
    <addaction name="actionProperties"/>
   </widget>
   <widget class="QMenu" name="menu_Plot">
    <property name="title">
     <string>&amp;Plot</string>
    </property>
    <addaction name="action_Parse_Log"/>
    <addaction name="action_Save_Log"/>
    <addaction name="action_Export_Data"/>
    <addaction name="action_Plot_Data"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Run"/>
   <addaction name="menu_Plot"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Open_2">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="text">
    <string>&amp;Save As</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Exit_2">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="action_Exit_3">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="text">
    <string>&amp;Cut</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="action_Paste_2">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="action_Joystick">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Joystick</string>
   </property>
  </action>
  <action name="action_Keyboard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Keyboard</string>
   </property>
  </action>
  <action name="action_Help_Dialogs">
   <property name="text">
    <string>&amp;Help Dialogs</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="action_Start_Elka">
   <property name="text">
    <string>&amp;Start Elka</string>
   </property>
  </action>
  <action name="action_Parse_Log">
   <property name="text">
    <string>&amp;Parse Log</string>
   </property>
  </action>
  <action name="action_Save_Log">
   <property name="text">
    <string>&amp;Save Log</string>
   </property>
  </action>
  <action name="action_Export_Data">
   <property name="text">
    <string>&amp;Export Data</string>
   </property>
  </action>
  <action name="action_Plot_Data">
   <property name="text">
    <string>&amp;Plot Data</string>
   </property>
  </action>
  <action name="actionProperties_2">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionStart_Elka">
   <property name="text">
    <string>Start Elka</string>
   </property>
  </action>
  <action name="action_Stop_Elka">
   <property name="text">
    <string>&amp;Stop Elka</string>
   </property>
  </action>
  <action name="action_Set_Gains">
   <property name="text">
    <string>&amp;Set Gains</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
